lib/zendesk-uploader/client-wrapper.js

```diff
@@ -1,16 +1,19 @@
 const request = require('request');
 const fs = require('fs');
+const Queue = require('promise-queue');
```

Comment:
Queue will use native promises by default. Since we're using `bluebird` I guess it might be a good idea not to mix it. Need to call `Queue.configure` somewhere in the beginning of the script.